Rakefile

```diff
@@ -35,3 +35,8 @@ task :spec do
   ruby "spec/unicode_emoji_spec.rb"
 end
 task default: :spec
+
+desc "#{gemspec.name} | Generates all regex constants and saves them to lib/unicode/emoji/{generated,generated_native} directories"
+task :generate_constants do
+  load "data/generate_constants.rb", true
```

Comment:
Passing `true` to load will "wrap" loaded code inside a temporary module. So for example, when loaded code make global `include` (like our script "generate_constants.rb") then it won't affect a global (main) object.